package.json

```diff
@@ -47,50 +47,49 @@
   "peerDependencies": {
     "ethers": ">=4.0.45"
```

Comment:
hum why do we have this ? also we moved to v5 :thinking:

Reply:
Ok shall we remove this?

There's a devDependency for v5 [here (line 80)](https://github.com/Open-Attestation/token-registry/pull/27/files/d63ebdc9c99b3c8d1e563266b5c52ffc6686249a#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R80).

Reply:
The thing is that dev dependencies are not installed, so it won't be available for consumers. I'm not sure of the reason the peer dependency is here though

Reply:
As requested, I've bumped the ethers peer dependency from v4.0.45 to v5.0.8